src/serviceWorker.ts

```diff
@@ -0,0 +1,149 @@
+// This optional code is used to register a service worker.
```

Comment:
Do we need service workers? I'm not familiar with service workers, but my understanding is they provide offline-first access. If my understanding is correct, how useful might that be if we're dynamically loading vector tiles via API?

Reply:
Don't expect we'll be using service workers, this all comes with CRA. Fine with leaving it in if you are.

Reply:
I'm fine with it. I don't think it'll impact number of users or time to ship over the long term.

I think getting a testing / staging pipeline up and running is really important for figuring out whether something works and minimizing time to ship, and I think generally having smaller diffs (among other things like linting) makes it easier for code reviewers to see bugs.